### ISSUE TITLE
Wrong icons

### ISSUE THREAD

**Issue by ziulev** [yamatsum/nvim-nonicons#7]
Something went wrong, could you help me to install the plugin?

I'm using macOS
1. Installed https://github.com/yamatsum/nonicons/blob/master/dist/nonicon.ttf
2. Installed plugin via Packer 
```
    use {
      'yamatsum/nvim-web-nonicons',
      requires = {'kyazdani42/nvim-web-devicons'}
    }
```
And it looks like this:
![image](https://user-images.githubusercontent.com/14956407/112012096-2ecc6800-8b29-11eb-87ff-054a16a17e88.png)

**Comment by yamatsum** [comment]
What terminal app are you using?
Did you set the font in the terminal app?

**Comment by ziulev** [comment]
Alacrity
Hm, no, I didn't 
I prefer to use inconsolata, is there any another way?

**Comment by yamatsum** [comment]
https://github.com/alacritty/alacritty/issues/3318
It seems that multiple fonts cannot be used in Alacrity, so there is no choice but to apply a patch

**Comment by ziulev** [comment]
How can I apply the patch? Do you mean to patch my current font or?

**Comment by yamatsum** [comment]
https://github.com/yamatsum/nonicons/issues/4
I don't prepare to patch font currently.

**Comment by ndavidq0** [comment]
I'm on kitty, I put the line `symbol_map U+f101-U+f208 nonicon` on kitty.conf. I'm using nvim-tree.lua, the file icons doesn't seem to be correct, however.
![NvimTree](https://user-images.githubusercontent.com/74260683/115075908-b96b7180-9ef3-11eb-8640-0b89c9e5a9e6.png)

**Comment by FrancoBregante** [comment]
Hi! I had the same issue with Kitty. I've just changed the line in the config file to `symbol_map U+f101-U+f208 nonicons`. It's changing nonicon to plural form. Hope it would work for you!

**Comment by vinodnimbalkar** [comment]
> Hi! I had the same issue with Kitty. I've just changed the line in the config file to `symbol_map U+f101-U+f208 nonicons`. It's changing nonicon to plural form. Hope it would work for you!

Thanks, It works for me.

**Comment by ndav1d** [comment]
it does not work. Some of the icons are wrong nonetheless.
![image](https://user-images.githubusercontent.com/74260683/124353773-635bbf80-dc00-11eb-97af-fab10000e88c.png)

**Comment by FrancoBregante** [comment]
@ndav1d the vimscript and folder icon seems good. Which one do you think is wrong? Also, please add the fonts setting in your kitty conf

**Comment by ndav1d** [comment]
In these icons we can already see some that are not correct... For example, a "bug" where it should be an opened empty folder haha.
![image](https://user-images.githubusercontent.com/74260683/124366405-8bbadc80-dc47-11eb-8cb7-277c68cf49f9.png)

My kitty font config is the following 
```
font_family      JetBrainsMono Nerd Font Mono
symbol_map U+f101-U+f208 nonicons
```

**Comment by FrancoBregante** [comment]
Oh... I see it now hahaha. I have the same issue, but as I set another icon for folders, I haven't realized it yet.

I know it's not the answer you are looking for, but you can set the icons like this:
`
vim.g.nvim_tree_icons = {
  folder = {
    default = "",
    open = "",
    empty = "",
    empty_open = "",
    symlink = "",
    symlink_open = "",
  },
}
`

**Comment by alvarogalloc** [comment]
In macos by using the patched font still doesn't works :(

**Comment by ndav1d** [comment]
yes the icons are not in the right place. it won't work on any device
